lib/livebook/application.ex

```diff
@@ -169,6 +170,13 @@ defmodule Livebook.Application do
     end
   end
 
+  defp load_lb_env_vars do
+    for {name, value} <- System.get_env(), String.starts_with?(name, "LB_") do
+      Livebook.Secrets.set_secret(%Livebook.Secrets.Secret{name: name, value: value})
```

Comment:
Once we do this, we will start storing the secret in the settings. I'm not sure if that's the expected behaviour, @josevalim?

Reply:
Also, we should remove the prefix before storing as app secret

Reply:
We want the secrets to appear for toggling in the sidebar but they should not be persisted in a way they would also be available when the Livebook app restarts and the secret is no longer available.